cli/command/formatter/stats.go

```diff
@@ -167,7 +167,7 @@ func (c *containerStatsContext) MemUsage() string {
 func (c *containerStatsContext) MemPerc() string {
 	header := memPercHeader
 	c.AddHeader(header)
-	if c.s.IsInvalid {
+	if c.s.IsInvalid || c.s.OSType == winOSType {
```

Comment:
@jhowardmsft, I added this part because I thought Windows doesn't collect memory percentage stats, is that right?